ams/ams-server/src/main/java/com/netease/arctic/ams/server/ArcticMetaStore.java

```diff
@@ -186,6 +186,12 @@ private static String getArcticHome() {
   }
 
   public static void startMetaStore(Configuration conf) throws Throwable {
+    //prepare env
+    if (conf.getString(ArcticMetaStoreConf.DB_TYPE).equals("derby")) {
+      DerbyService derbyService = new DerbyService();
+      derbyService.createTable();
+    }
+
```

Comment:
This may execute createTable at each start. and derby DDL can't be executed for mulit times.